### ISSUE TITLE
shop/shop.tar.gz does not exist

### ISSUE THREAD

**Issue by peterjaap** [mwr/magedeploy2#4]
I've generated my artifacts, but when I run `server:setup local`, it complains that it cannot find the tar.gz file;

```
➜  magedeploy2-test./vendor/bin/dep server:setup local
➤ Executing task deploy:initialize
Deploying 20170317050054 to releases/20170317050054
✔ Ok
✔ Executing task deploy:prepare
✔ Executing task build:shared_dirs_generate
✔ Executing task deploy:release
➤ Executing task build:upload_artifacts


  [RuntimeException]
  Uploading path 'shop/shop.tar.gz' does not exist.


server:setup [-p|--parallel] [--no-hooks]
```

**Comment by peterjaap** [comment]
Ah, it's in artifacts. To test I moved it from shop/ to artifacts/ and ran it again;

```
➤ Executing task deploy:initialize
Deploying 20170317050228 to releases/20170317050228
✔ Ok
✔ Executing task deploy:prepare
✔ Executing task build:shared_dirs_generate
✔ Executing task deploy:release
➤ Executing task build:upload_artifacts
Upload file shop/shop.tar.gz to /Users/peterjaap/Sites/mijnheer2/releases/20170317050228/shop.tar.gz

  [RuntimeException]
  Uploading path 'shop/pub_static.tar.gz' does not exist.


server:setup [-p|--parallel] [--no-hooks]
```

That seems to go a little bit better. Somewhere it's hardcoded I guess.

**Comment by mwr** [comment]
For a moment I was wondering how that could be, because I recently updated that part while introducing the artifacts dir and making it configurable.
n98-deployer had to be updated as well to pick up the artifacts from the artifacts-dir.
And shop was indeed hardcoded back then.
Long story short: it is fixed in the latest develop with n98-deployer repository. 
https://github.com/netz98/n98-deployer/commit/92acc72049965d3d09a30f86b8776c8ec4cdb7e3
https://github.com/netz98/n98-deployer/commit/2ad2f48ba2669c66d715780dae19f5e72cf943fe

Turns out I forgot create a new release to make the new version for n98-deployer public.
So in version 1.1.0 of n98-deployer the upload uses the artifacts dir to find the tar.gz
